web/css/timeline.css

```diff
@@ -560,3 +564,44 @@ g.plot rect.data-bar-invisible {
 #timeline-hide:hover .hamburger path {
   fill: #1a1a1a;
 }
+#timeline-footer .ab-group-case text {
+  font-size: 20px;
+}
+.ab-group-case {
+  display: none;
+}
+.ab-group-case-active {
+  fill: #fff;
+}
+.ab-group-case polygon,
+.ab-group-case path,
+.ab-group-case rect {
+  fill: #808080;
+  opacity: 0.7;
+}
+.ab-group-case-active path {
+  stroke: #515151;
+  stroke-width: 1;
+  stroke-opacity: 1;
+}
+.ab-group-case-active polygon,
+.ab-group-case-active path,
+.ab-group-case-active rect {
+  fill: #fff;
+  opacity: 1;
+  pointer-events: initial;
+  cursor: pointer;
+}
+.ab-group-case text {
+  fill: #fff;
+  pointer-events: none;
+}
+.ab-group-case-active text {
+  fill: #000;
+}
+.ab-active .ab-group-case {
+  display: block;
+}
+.ab-active #guitarpick {
+  display: none !important;
```

Comment:
Is there way to avoid the `!important` use here?

Reply:
There must be a way! good call